### PR TITLE
CI improvements

### PR DESCRIPTION
**Pull request by tamird**
@localshred @abrandoned @liveh2o

### CODE REVIEW

File: install-protobuf.sh [hunk 1]
```diff
@@ -1,8 +1,10 @@
+#!/usr/bin/env sh
+
 set -ex
 
-basename=protobuf-2.6.0
-tarball=$basename.tar.bz2
+version=2.6.1
```
Comment (localshred) [inline install-protobuf.sh]: You could easily take an arg here and default to 2.6.1:

```bash
version=${1:-"2.6.1"}
```

Thus you could install whichever version you wish without having to update the script.

Comment (tamird) [inline install-protobuf.sh]: we'd still have to update it to move the default along to keep up with latest. not sure i see the value

**Comment by tamird** [comment]
ping! happy to close this if y'all aren't interested

**Comment by tamird** [comment]
@localshred this adds testing with protobuf 3 (currently fails, but still)

**Comment by tamird** [comment]
@localshred shall I close this?

**Pull request merged**